test/indexer/data/1b6d8348-d6e9-406a-aa6a-7ee886e52bf9.tdr.hca.json

```diff
@@ -7,8 +7,8 @@
             "indexed": false,
             "content-type": "whatever format these are in; dcp-type=data",
             "version": "2019-09-24T09:35:06.958773Z",
-            "crc32c": "",
-            "sha256": "",
+            "crc32c": "00000000",
```

Comment:
These changes should be in a separate, non-split commit. We'd like to keep them, even if we revert the changes that assert the schema location.

Reply:
By these changes, I mean all changes to cans.